docker-base-services.yml

```diff
@@ -6,50 +6,50 @@
 #
 ---
 services:
-  fam-database:
-    image: postgres:14.1-alpine
-    mem_limit: 128m
-    environment:
-      - POSTGRES_USER=postgres
-      - POSTGRES_PASSWORD=postgres
-      - POSTGRES_DB=fam
-      - POSTGRES_HOST_AUTH_METHOD=password
-    healthcheck:
-      test: ["CMD-SHELL", "pg_isready -U postgres"]
-      interval: 5s
-      timeout: 5s
-      retries: 6
-    networks:
-      - fam
+    fam-database:
+        image: postgres:17.4-alpine
```

Comment:
Why is this 17 when AWS is set to 16 below?

Reply:
I tried 17.4 on docker first and it worked locally but until I found out 17.4 is not available for AWS aurora-postgresql yet when deployment failed (version not found error). I leave it there in docker thought it didn't hurt to have local a bit higher version. Maybe you might prefer they are the same version?

Reply:
I'd keep it the same version.

Reply:
It is the same now.